src/lesson-2/functions/checkIsDoneBeforeDeadline.ts

```diff
@@ -0,0 +1,17 @@
+import { data } from '../parsedData';
+
+export const checkIsDoneBeforeDeadline = (taskId: string): boolean | null => {
+  const task = data.find((item) => item.id === taskId);
+
+  if (task) {
+    const today = Date.now();
+    const deadline =
```

Comment:
new Date() приймає як string, так і обʼєкт, тому можна упростити:
```suggestion
    const today = new Date() 
    const deadline = new Date(task.deadline)
```